connectors/src/connectors/github/temporal/activities.ts

```diff
@@ -153,10 +157,152 @@ export async function githubUpsertIssueActivity(
       repoId: repoId.toString(),
       issueNumber,
       connectorId: connector.id,
+      isDiscussion: false,
     });
   }
 }
 
+export async function githubUpsertDiscussionActivity(
+  installationId: string,
+  repoName: string,
+  repoId: number,
+  login: string,
+  discussionNumber: number,
+  dataSourceConfig: DataSourceConfig,
+  loggerArgs: Record<string, string | number>
+) {
+  const localLogger = logger.child({
+    ...loggerArgs,
+    discussionNumber,
+  });
+
+  localLogger.info("Upserting GitHub discussion.");
+
+  const discussion = await getDiscussion(
+    installationId,
+    repoName,
+    login,
+    discussionNumber
+  );
+
+  let renderedDiscussion = `# ${discussion.title}||\n${discussion.bodyText}||\n`;
+
+  let nextCursor: string | null = null;
+
+  for (;;) {
+    const { cursor, comments } = await getDiscussionCommentsPage(
+      installationId,
+      repoName,
+      login,
+      discussionNumber,
+      nextCursor
+    );
+
+    for (const comment of comments) {
+      renderedDiscussion += `${comment.author.login}: ${comment.bodyText}||`;
+      let nextChildCursor: string | null = null;
+      for (;;) {
+        const { cursor: childCursor, comments: childComments } =
+          await getDiscussionCommentRepliesPage(
+            installationId,
+            comment.id,
+            nextChildCursor
+          );
+        for (const childComment of childComments) {
+          renderedDiscussion += `----${childComment.author.login}: ${childComment.bodyText}||`;
+        }
+
+        if (!childCursor) {
+          break;
+        }
+
+        nextChildCursor = childCursor;
+      }
+    }
+
+    if (!cursor) {
+      break;
+    }
+
+    nextCursor = cursor;
+  }
+
+  const documentId = getDiscussionDocumentId(
+    repoId.toString(),
+    discussionNumber
+  );
+  const tags = [
```

Comment:
maybe more tags could be helpful. Would `isAcceptedAnswer` be useful here ?

Reply:
Don't we have the usual lastUpdatedTs ?

Reply:
isAcceptedAnswer seems like a good candidate indeed.

Reply:
Indeed, forgot the last updated TS. Adding both 👍

Reply:
Actually `isAcceptedAnswer` doesn't really make any sense as a tag, as it applies to a specific comment and not the discussion as a whole. 

Maybe I should instead preprend `"[ACCEPTED ANSWER]"` to the comment if it is the accepted answer

Reply:
Sounds reasonable :+1: